src/checks/y_check_constants_interface.clas.testclasses.abap

```diff
@@ -54,18 +54,19 @@ ENDCLASS.
 CLASS ltc_empty_interface DEFINITION INHERITING FROM ltc_constants_only FOR TESTING RISK LEVEL HARMLESS DURATION SHORT.
   PROTECTED SECTION.
     METHODS get_code_without_issue REDEFINITION.
+  PRIVATE SECTION.
+
 ENDCLASS.
 
 CLASS ltc_empty_interface IMPLEMENTATION.
 
   METHOD get_code_without_issue.
     result = VALUE #(
       ( 'REPORT y_example.' )
-
+      ( '"has a comment ' ) " #562
```

Comment:
Please don't add "edit history" comments like this - the git history in the repository will already show the pull request number, there is no need to duplicate it in the code.